editor/export/export_template_manager.cpp

```diff
@@ -512,7 +519,8 @@ bool ExportTemplateManager::_install_file_selected(const String &p_file, bool p_
 
 		String file = file_path.get_file();
 
-		if (file.size() == 0) {
+		// Skip the __MACOSX directory created by macOS's built-in file zipper.
+		if (file.size() == 0 || file.begins_with("__MACOSX")) {
```

Comment:
```suggestion
		if (file.is_empty() || file.begins_with("__MACOSX")) {
```

Reply:
The suggestion makes sense, thanks!

This probably something for a different PR, but I feel like it might make sense to rename the `file` variable to `filename` or `filepath`; the naming right now makes it sound like we'd be checking the *contents* of the file to see if it is empty or starts with a given string, when in fact we're just looking at the *path* in the ZIP file.